x-pack/plugins/enterprise_search/public/applications/test_helpers/enzyme_rerender.ts

```diff
@@ -5,13 +5,13 @@
  * 2.0.
  */
 
-import { ShallowWrapper } from 'enzyme';
+import { CommonWrapper } from 'enzyme';
 
 /**
  * Quick and easy helper for re-rendering a React component in Enzyme
  * after (e.g.) updating Kea values
  */
-export const rerender = (wrapper: ShallowWrapper) => {
+export const rerender = (wrapper: CommonWrapper) => {
```

Comment:
Does this change need a broader consensus from the Workplace team?

Reply:
It shouldn't, I'm just broadening this so it accepts the `ReactWrapper` type which is generated by our `mountWithIntl` helper function.  This only affects our tests so impact is minimal as well.